jekyll/_cci2/release/set-up-a-release-environment.adoc

```diff
@@ -0,0 +1,111 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Set up a release environment - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to set up a new CircleCI release environment and install the CircleCI release agent.
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**. During open preview release features are available for free. Charges will be associated with release features once they are generally available.
+
+Visualise and control deployments to your Kubernetes cluster with CircleCI release environments. The steps outlined on this page guide you to set up a release environment and install the CircleCI release agent into your Kubernetes cluster.
+
+[#introduction]
+== Introduction
+
+In this tutorial you will set up an integration between CircleCI and your Kubernetes cluster. Once configured, release status can be reported to the CircleCI web app, and the following commands can be executed in your cluster by the release agent via controls in the CircleCI UI:
+
+* Restore a version
+* Scale a version
+* Restart a version
+* Retry a release (when using Argo Rollouts)
+* Promote a release (when using Argo Rollouts)
+* Cancel a release (when using Argo Rollouts)
+
+[#prerequisites]
+== Prerequisites
+
+In order to set up a release environment with CircleCI you will need to ensure the following prerequisites are fulfilled:
+
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
+* A Kubernetes cluster
+* A project building on CircleCI that deploys to your Kubernetes cluster. See the xref:../create-project#[Create a project] page for steps.
+* You must have write access to the project associated with the component being deployed. For full details of roles and permissions for GitLab and GitHub App integrations, see the xref:../roles-and-permissions-overview.adoc#[Roles and permissions overview]. If your integration is through Bitbucket or the GitHub OAuth app, you will need to be an org admin in Bitbucket/GitHub respectively. To find out which GitHub integration you have, see the xref:../github-apps-integration.adoc#[GitHub App integration] page
+
+CircleCI supports Kubernetes, Helm, and Argo Rollouts. The supported versions are listed below:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+|System
+|Supported versions
+
+|Kubernetes
+a| 1.28.0 +
+1.27.0
+
+|Helm
+a| 3.12.0 +
+3.11.0
+
+|Argo Rollouts (optional)
+a| 1.6.0 +
+1.5.0
+|===
+
+[#create-release-environment]
+== 1. Create your release environment and token
+
+. In the CircleCI web app, click **Releases** in the sidebar
+. Click btn:[Add Release Environment]
+. Enter a name for your release environment and click btn:[Create Environment]
+. Click to select the environment you just created
```

Comment:
Where do I click to select? After crearting my env, I see this again: 
![image](https://github.com/circleci/circleci-docs/assets/4623/dccc9a31-a5ab-4382-b738-3af24fec64d2)

Reply:
When I create a new environment, I end up here:
<img width="988" alt="Screenshot 2023-10-10 at 22 15 42" src="https://github.com/circleci/circleci-docs/assets/24589403/b03b5918-3ed2-4010-8fb7-151a44b02e43">